pkg/storaged/crypto-keyslots.jsx

```diff
@@ -560,9 +562,15 @@ function edit_passphrase_dialog(block, key) {
                       { validate: val => !val.length && _("Passphrase cannot be empty") }),
             Skip("medskip"),
             PassInput("new_passphrase", _("New passphrase"),
-                      { validate: val => !val.length && _("Passphrase cannot be empty") }),
+                      {
+                          validate: val => !val.length && _("Passphrase cannot be empty"),
+                          new_password: true
+                      }),
             PassInput("new_passphrase2", _("Repeat passphrase"),
-                      { validate: (val, vals) => vals.new_passphrase.length && vals.new_passphrase != val && _("Passphrases do not match") })
+                      {
+                          validate: (val, vals) => vals.new_passphrase.length && vals.new_passphrase != val && _("Passphrases do not match"),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18906-20230616-104507-5b7c7f1b-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)